.gitlab-ci.yml

```diff
@@ -1185,6 +1187,24 @@ deploy_puppy:
     - export PACKAGE_VERSION=$(inv agent.version --url-safe)
     - aws s3 cp --region us-east-1 ./agent $S3_DSD6_URI/puppy/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
 
+# deploy process agent and system-probe to staging bucket
+deploy_process_and_sysprobe:
```

Comment:
`s/deploy/upload`?

Reply:
The other steps are using `deploy` I think it's better to keep it consistent